GameData/RealismOverhaul/Engine_Configs/HG-3_Config.cfg

```diff
@@ -0,0 +1,142 @@
+// HG-3 series engine
+// SXT
+//  ==================================================
+//  Rocketdyne HG-3 engine.
+
+//  Dimensions: 2.1 m x 3.5 m
+//  Gross Mass: ?? (Assumed 1780 kg as reough estimate)
+//  Throttle Range: 77% to 100%
+//  Burn Time: 600 s
+//  O/F Ratio: ?? (Assumed 5.5 ala J-2)
+
+//  Source 1: http://www.astronautix.com/h/hg-3.html
+//  Source 2: http://www.astronautix.com/h/hg-3-sl.html
+//  Source 3: https://history.msfc.nasa.gov/saturn_apollo/propulsion_center.html
+//  ==================================================
+@PART[*]:HAS[#engineType[HG3]]:FOR[RealismOverhaulEngines]
+{
+	@title = HG-3 Series
+	%manufacturer = Rocketdyne
+	@description = Rocketdyne HG-3 engine, which served as a stepping stone between the J-2 and RS-25. Ensure you enable roll capability or disable gimbal as required for your application. Diameter: [2.01 m].
+	
+	MODULE
+	{
+		name = ModuleEngineConfigs
+		type = ModuleEngines
+		configuration = J-2
+		@configuration:NEEDS[RP-0] = J-2-200klbf
```

Comment:
configuration needs changing to HG-3

and the `@configuration:NEEDS` line can probably go